source/ConditionSet.h

```diff
@@ -29,7 +29,7 @@ class DataWriter;
 class ConditionSet {
 public:
 	using Conditions = std::map<std::string, int64_t>;
-	ConditionSet() = default;
+	ConditionSet() noexcept = default;
```

Comment:
This will cause a compile error in the steam environment due to the throw specification being different than the implicit one.

(It is different because of the `std::string` class field types.)
```suggestion
	ConditionSet() = default;
```